vars/provisionNodes.groovy

```diff
@@ -120,14 +120,26 @@ def call(Map config = [:]) {
                               chmod 600 /localhome/jenkins/.ssh/{authorized_keys,id_rsa*}
                               chown -R jenkins.jenkins /localhome/jenkins/.ssh
                               echo \\"jenkins ALL=(ALL) NOPASSWD: ALL\\" > /etc/sudoers.d/jenkins
-                              yum install -y epel-release
-                              yum install -y openmpi CUnit fuse python36-PyYAML python36-nose    \
-                                             python36-pip valgrind python36-paramiko             \
-                                             python2-avocado python2-avocado-plugins-output-html \
-                                             python2-avocado-plugins-varianter-yaml-to-mux       \
-                                             python-debuginfo python2-aexpect libcmocka          \
-                                             python-pathlib python2-numpy git                    \
-                                             golang-bin
+                              yum -y install epel-release
+                              if ! yum -y install openmpi CUnit fuse
+                                                  python36-PyYAML              \
+                                                  python36-nose                \
+                                                  python36-pip valgrind        \
+                                                  python36-paramiko            \
+                                                  python2-avocado              \
+                                                  python2-avocado-plugins-output-html \
+                                                  python2-avocado-plugins-varianter-yaml-to-mux \
+                                                  python-debuginfo             \
+                                                  python2-aexpect libcmocka    \
+                                                  python-pathlib python2-numpy \
+                                                  git golang-bin; then
+                                  rc=\\${PIPESTATUS[0]}
+                                  for file in /etc/yum.repos.d/*.repo; do
```

Comment:
You could replace this whole block with "grep . /etc/yum.repos.d/*.repo"

Reply:
Good point.  I have used that idiom in the past.